pkg/kubelet/images/image_gc_manager.go

```diff
@@ -338,13 +338,29 @@ func (im *realImageGCManager) freeSpace(bytesToFree int64, freeTime time.Time) (
 	im.imageRecordsLock.Lock()
 	defer im.imageRecordsLock.Unlock()
 
+	// Make the ListImages into a map to grab an image by ID
+	allImages, err := im.runtime.ListImages()
+	if err != nil {
+		return 0, err
+	}
+	imagesMap := make(map[string]container.Image)
```

Comment:
nit: Preallocate the capacity of the map? - Negligible gain in most cases, but hosts can end up with a lot of images so it's probably worth it

Reply:
how big should I make it ?

Reply:
fixed!